nvflare/apis/impl/job_def_manager.py

```diff
@@ -103,8 +107,9 @@ def create(self, meta: dict, uploaded_content: bytes, fl_ctx: FLContext) -> Dict
         meta[JobMetaKey.STATUS.value] = RunStatus.SUBMITTED.value
 
         # write it to the store
+        stored_data = {JobDataKey.JOB_DATA.value: uploaded_content, JobDataKey.WORKSPACE_DATA: None}
```

Comment:
Should we put `JobDataKey.WORKSPACE_DATA` here though?

Since each job config can be run multiple times, the workspace_data or result_data should be store in another way?

It should not be 1-to-1 mapping to the JOB config?

Reply:
No, we have decided that one job only run once. This is 1-to-1 mapping.